package.json

```diff
@@ -305,7 +305,8 @@
     "ts-dedent": "2.0.0",
     "ts-jest": "26.1.4",
     "ts-node": "9.0.0",
-    "typescript": "3.9.7"
+    "typescript": "3.9.7",
+    "babel-jest": "26.2.2"
```

Comment:
i am curious why this is here

Reply:
This just came when I added the project with nx.